src/conversions.rs

```diff
@@ -600,6 +600,7 @@ impl<C: Clone, T: FromJSValConvertible<Config=C>> FromJSValConvertible for Vec<T
         let mut iterator = ForOfIterator {
             cx_: cx,
             iterator: RootedObject::new_unrooted(),
+            nextMethod: RootedValue::new_unrooted(),
             index: ::std::u32::MAX, // NOT_ARRAY
```

Comment:
On the android cross-compile for mac, this has an additional `__bindgen_padding_0` field.

Reply:
Oh goody, a `cfg` that depends on both the source and the target. Do you know what its type is?

Reply:
It's a u32.

Reply:
I put it in when the target is android, hopefully it doesn't depend on the host OS too.

Reply:
The `__bindgen_padding_0` was removed again, and its lack is breaking my build when I build the smup Servo branch.

Reply:
Specifically, 694bc50  should be removed from this pull request to fix this problem.